src/components/tagApplyMenu/AposTagApply.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div class="apos-apply-tag-menu">
+    <AposContextMenu :tipAlignment="tipAlignment">
+      <div class="apos-apply-tag-menu__inner">
+`        <AposStringInput v-on:input="updateSearchInput" 
+          :field="searchField" :value="searchValue" :status="searchStatus" ref="textInput"
+        />
+        <div class="apos-apply-tag__create">
+          <AposButton 
+            v-on:click="create" 
+            :label='createLabel'
+            type="quiet"
+            :disabled="disabledCreate"
+          />
+        </div>
+        <ol v-if="tags" class="apos-apply-tag-menu__tags">
+          <transition-group name="fade" tag="li">
+            <li class="apos-apply-tag-menu__tag" v-for="tag in searchTags" :key="tag.slug">
+              <AposCheckbox 
```

Comment:
Something isn't getting connected right. When I create a new tag, the checkbox doesn't work as expected. 

![jeepers](https://user-images.githubusercontent.com/1155984/82075470-e8ed7a80-96a1-11ea-8e35-a62789a5d0ee.gif)

Reply:
for now going with regenerating the component keys on click interactions, which forces them to refresh keeping them in sync w the state